src/utils/dif.rs

```diff
@@ -335,11 +335,28 @@ impl<'a> DifFile<'a> {
         match self {
             DifFile::Archive(archive) => {
                 let mut features = ObjectDifFeatures::none();
-                for object in archive.get().objects().filter_map(Result::ok) {
+
+                let mut add_object_features = |object: &Object| {
                     features.symtab = features.symtab || object.has_symbols();
                     features.debug = features.debug || object.has_debug_info();
                     features.unwind = features.unwind || object.has_unwind_info();
                     features.sources = features.sources || object.has_sources();
+                };
+
+                for object in archive.get().objects().filter_map(Result::ok) {
+                    add_object_features(&object);
+
+                    // Combine features with an embedded Portable PDB, if any.
```

Comment:
the reason we are doing this is because otherwise the object does not have any features at all, and we would not upload it at all?

Reply:
Oh, I saw the new testcase that should report sources in `dif check`

Reply:
yes, but that gave me another idea... I have to check whether this recent change doesn't cause the .dll itself to be uploaded now (uselessly)

Reply:
ok, works fine (not do what I was afraid of). Updated the comment to clarify why it's here.